README.md

```diff
@@ -98,6 +98,9 @@ kubectl apply -f deploy/csidriver-1.15.yaml
 
 # [OPTIONAL] To deploy driver on windows nodes
 kubectl apply -f deploy/secrets-store-csi-driver-windows.yaml --namespace $NAMESPACE
+
+# [OPTIONAL] Install RBAC bindings to allow secret syncing
+kubectl apply -f deploy/rbac-secretprovidersyncing.yaml
```

Comment:
I'm not sure if i should update this now or after there is a copy from `manifest_staging` into `deploy` - probably after?

Reply:
I think we should probably do it after, when we cut a release.